apps/web/pages/export.tsx

```diff
@@ -0,0 +1,253 @@
+import {
+  Button,
+  Card,
+  CircularProgress,
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+} from '@usevenice/ui'
+
+import {
+  CheckCircleFilledIcon,
+  CodeIcon,
+  CopyTextIcon,
+  DownloadIcon,
+  SyncIcon,
+} from '@usevenice/ui/icons'
+
+import type {GetServerSideProps} from 'next'
+import Link from 'next/link'
+import {useEffect, useState} from 'react'
+import {PreviewResult, usePreviewQuery} from '../components/export'
+import {ExternalLink} from '../components/ExternalLink'
+import {PageHeader} from '../components/PageHeader'
+import {PageLayout} from '../components/PageLayout'
+
+const PREVIEW_LIMIT = 8
```

Comment:
We likely want a label that says e.g. `8 of 200 shown`, or `8 results shown out of 200 total results total` since this could be confusing as to why only a small number are showing

I'll mock it out

Reply:
💯 implemented 37c433c